assets/sass/v2/_password.scss

```diff
@@ -0,0 +1,12 @@
+.password-authenticator {
+  &__rules-heading {
+    margin-bottom: 1rem;
+  }
+
+  &__rules-list {
+    margin-bottom: 1rem;
+    li {
```

Comment:
Highly prefer not using direct html elements in CSS. Consider moving to a class name.